using-mobile-extensions/adobe-audience-manager/migration.md

```diff
@@ -0,0 +1,141 @@
+# Migrating to AEPAudience reference
+
+This document is a reference comparison of AEPAudience (3.x) APIs against their equivalent ACPAudience (2.x) APIs.
+
+The AEPAudience extension is implemented purely in Swift and is compatible with the AEPCore Swift SDK. To ensure a smooth transition from the ACPAudience SDK, there are no major changes on the API names or definition. For more details, follow the migration guide below for your Swift or Objective-C mobile application.  If explanation beyond showing API differences is necessary, it will be captured as an info hint within that API's section.
```

Comment:
Random double space wormed its way in here. We can get rid of that. (Between "mobile application." and "If explanation...")

Reply:
Updated @calebk1. Please review when you have time. 
If it is good, can you help to merge the PR?